wordlist.txt

```diff
@@ -249,6 +266,8 @@ getPayloadV
 getters
 ghost
 Gilles
+Goeril
```

Comment:
Kinda sus :)

Reply:
I couldn't locate any docs that has these misspelled words. Perhaps, someone put it there to get around the spellcheck but then later corrected the grammar in the page.